ragatouille/models/colbert.py

```diff
@@ -361,68 +324,31 @@ def index(
             Path(self.run_config.root) / Path(self.run_config.experiment) / "indexes"
         )
 
-        self._write_collection_files_to_disk()
+        self.pid_docid_map = pid_docid_map
 
-        print("Done indexing!")
+        # inverted mapping for returning full docs
+        self.docid_pid_map = defaultdict(list)
```

Comment:
Should this be a call to `_save_index_metadata` instead? Since I don't think `index` saves collection.json or other metadata files to disk currently.

Reply:
I think I'm missing something here, understanding-wise.

Reply:
In my understanding, `_save_index_metadata` gets called at the very end of this (L347), and `_save_index_metadata` also writes the collection to disk, so it should be fine?

Reply:
Yep, my mistake, I was skimming to quickly. Thanks!